Validator/Constraints/EmbeddedAddressFormatConstraint.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Daften\Bundle\AddressingBundle\Validator\Constraints;
+
+use CommerceGuys\Addressing\Validator\Constraints\AddressFormatConstraint;
+use CommerceGuys\Addressing\Validator\Constraints\AddressFormatConstraintValidator;
+
+/**
+ * @Annotation
+ *
+ * @codeCoverageIgnore
+ */
+class EmbeddedAddressFormatConstraint extends AddressFormatConstraint
```

Comment:
Updating the README to have that validation constraint in the example field would probably be a good thing :-)